### PR TITLE
Adding chromatic stories

### LINKED ISSUE #1957: Add more chromatic stories
# 🙋 Feature Request

- See what components are missing chromatic stories
  - Compile a list for posterity:
  - Pick up a couple to write stories for (choose ones that have a bunch of different styles?)

EDIT: 
List of packages that don't have chromatic stories at all:
- Color components
- Combobox
- Form (maybe this one doesn’t need chromatic?)
- Icon (maybe this one doesn’t need chromatic?)
- Label
- Listbox
- Listview
- Menu
- Overlay
- Table

## 🧢 Your Company/Team

RSP

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes #1957
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/color/chromatic/ColorField.chromatic.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+let states = [
+  {isQuiet: true},
+  {isDisabled: true},
+  {validationState: ['valid', 'invalid']},
+  {isRequired: true},
+  {necessityIndicator: 'label'}
+];
+
+// Generate a powerset of the options
+let combinations: any[] = [{}];
```
Comment (snowystinger) [inline packages/@react-spectrum/color/chromatic/ColorField.chromatic.tsx:30]: We should really pull this out into a helper function, if you have the chance

File: packages/@react-spectrum/combobox/chromatic/ComboBoxAR.chromatic.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Meta} from '@storybook/react';
+
+// Original ComboBox chromatic story was too large to be processed
+const meta: Meta = {
+  title: 'ComboBoxAR',
```
Comment (snowystinger) [inline packages/@react-spectrum/combobox/chromatic/ComboBoxAR.chromatic.tsx:17]: would it make more sense to name it ComboBoxRTL? i spent a while trying to figure out what AR was

File: packages/@react-spectrum/listbox/chromatic/ListBox.chromatic.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  AlignLeft,
+  AlignRight,
+  Blower,
+  Book,
+  Copy,
+  Cut,
+  Paste
+};
+
+let hardModeProgrammatic = [
```
Comment (snowystinger) [inline packages/@react-spectrum/listbox/chromatic/ListBox.chromatic.tsx:39]: side note, i love that we've apparently actually settled on calling this hard mode... haha

File: packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx [hunk 4]
```diff
@@ -24,7 +24,7 @@ let states = [
   {isDisabled: true},
   {isReadOnly: true},
   {hideStepper: true},
-  {validationState: ['valid', 'invalid', undefined]}
+  {validationState: ['valid', 'invalid']}
```
Comment (snowystinger) [inline packages/@react-spectrum/numberfield/chromatic/NumberField.chromatic.tsx:27]: o, i doubled up on validationState undefined huh?

File: packages/@react-spectrum/color/chromatic/ColorField.chromatic.tsx (hunk 1 above)
Comment (LFDanLu) [reply packages/@react-spectrum/color/chromatic/ColorField.chromatic.tsx:30]: good idea, I'll see if I can do that.

File: packages/@react-spectrum/tooltip/chromatic/TooltipTrigger.chromatic.tsx [hunk 5]
```diff
@@ -22,13 +22,12 @@ const meta: Meta<SpectrumTooltipTriggerProps> = {
   parameters: {
     chromaticProvider: {colorSchemes: ['light'], locales: ['en-US'], scales: ['medium'], disableAnimations: true},
     // chromatic needs a bit more time than disableAnimations allows
-    chromatic: {delay: 300}
+    chromatic: {delay: 2000}
```
Comment (LFDanLu) [inline packages/@react-spectrum/tooltip/chromatic/TooltipTrigger.chromatic.tsx:25]: An effort to make the chromatic test less flakey

**Review by dannify — APPROVED** [review]

**Review by snowystinger — DISMISSED** [review]
Looks good, please run chromatic on main first, then merge this, then run chromatic one more time on main after merging

**Review by dannify — APPROVED** [review]

**Review by snowystinger — APPROVED** [review]
reapproving

**Pull request merged**